app/Http/Controllers/RandomizeController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Mismatch;
+use App\Services\StatsdAPIClient;
+
+class RandomizeController extends Controller
+{
+    use Traits\ReviewMismatch, Traits\StatsTracker;
+
+    /**
+     * Instantiate a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct(StatsdAPIClient $statsd)
+    {
+        $this->middleware('simulateError');
+        $this->statsd = $statsd;
+    }
+
+    public function index()
+    {
+        $itemIds = Mismatch::with('importMeta.user')
+            ->where('review_status', 'pending')
+            ->whereHas('importMeta', function ($import) {
+                $import->where('expires', '>=', now());
+            })
+            ->inRandomOrder()
+            ->limit(15)
+            ->pluck('item_id')->toArray();
+        
+        if (count($itemIds) > 0) {
+            $itemIdsToString = implode('|', $itemIds);
+
+            return redirect()->action(
+                [ResultsController::class, 'index'],
+                ['ids' => $itemIdsToString]
+            );
+        }
+    }
```

Comment:
And what happens otherwise? What does Laravel do if we don’t return anything?

Reply:
This is still to be implemented. I will put that in a separate PR since it includes creating a view to display the message saying that there are no matching items and it would make this PR too big. If you think it should go in this one i could it here.

Reply:
> And what happens otherwise? What does Laravel do if we don’t return anything?

I have added this now in a new commit.

Reply:
It looks like Laravel just returns an empty page in that case, which seems fine for now. (We decided in Mattermost to leave the “no results” part to a second pull request.)